### DISCUSSION TITLE
Kraken and season countdown board on the Beta branch

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#307] [Announcements]
I just pushed and update on the Beta branch. It contains some fixes and the new countdown feature (based on the one kempdawg20  made).  

IMPORTANT: If you already had the latest version installed, changing branch and using the Beta version should not have any issues. If not, depending on which version you are running, you might have to reconfigure your scoreboard as some stuff have been added overtime.

If you want to get on the Beta branch, connect (SSH) to your scoreboard and run those commands one by one.

```
cd nhl-led-scoreboard
git checkout beta
git pull
```
At the moment the new features are not added in the nhl_setup app. So you will have to add them manually in your config file. To open it, from the root of nhl-led-scoreboard (you should be there after running the command above), run this command (DO NOT USE SUDO).

```
nano config/config.json
```

To use the season countdown board, simply add `seasoncountdown` to the off_day state and which ever you want. 
If you want to add the Kraken as your favorite team, simply add `Kraken` in the `"teams"` list. 

Once you are done editing your config file, do those steps

```
CTRL+X
Y
ENTER
```

Then reboot your scoreboard

`sudo reboot -n`

I wont make an official release at least until the pre-season started and a few games have been played. I expect the Interns at the NHL HQ will be working half time and we will have some weird and/or missing data coming from the API so, hold on to your tuques. 

EDIT: 
The new features are subject to change.

**Comment by kempdawg20** [comment]
Love the tweaks you made to it!

**Reply by riffnshred** [reply]
Thanks !!!
